auto-resource-redux/src/LogisticManager.lua

```diff
@@ -153,6 +155,15 @@ local function handle_items_request(storage, force, entity, item_requests)
         break
       end
     end
+
+    -- if there is still some left, just insert it anywhere in the network
```

Comment:
I think we'd want to check what the network has available and only insert what's necessary, to avoid inserting more items each time the alert is handled.

Reply:
Yeah, there is quite a lag with those alerts. Maybe it would be better to track ghosts instead of looking at alerts.  Although that wouldn't help with repair packs.